modules/api-svc/src/config.js

```diff
@@ -87,6 +87,7 @@ module.exports = {
                 cert: env.get('OUT_CLIENT_CERT_PATH').asFileContent(),
                 key: env.get('OUT_CLIENT_KEY_PATH').asFileContent(),
             },
+            useCertFilesForAuth: env.get('OUT_USE_CERT_FILES_FOR_AUTH').default('false').asBool()
```

Comment:
How about moving this paramter to wso2.auth.mTlsEnabled? And environment variable name "OAUTH_MUTUAL_TLS_ENABLED"

Reply:
updated